types/react-native/globals.d.ts

```diff
@@ -35,7 +35,21 @@ declare interface WindowOrWorkerGlobalScope {
     fetch(input: RequestInfo, init?: RequestInit): Promise<Response>;
 }
 
-interface Blob {}
+interface Blob {
+    readonly size: number;
+    readonly type: string;
+    slice(start?: number, end?: number): Blob;
+}
+
+interface BlobOptions {
+    type: string;
+    lastModified: number;
+}
+
+declare var Blob: {
+    prototype: Blob;
+    new (blobParts?: Array<Blob | string>[], options?: BlobOptions): Blob;
```

Comment:
I think there might be an error here, as this constructor expects an array of `String` or `Blob`, not an array of arrays, right? -> [https://developer.mozilla.org/en-US/docs/Web/API/Blob/Blob](https://developer.mozilla.org/en-US/docs/Web/API/Blob/Blob)

```
   new (blobParts?: (Blob | string)[], options?: BlobOptions): Blob;
```

Reply:
yes...I will send a PR to fix this.

Reply:
@maltoze - I think `options` being of type `BlobOptions` puts it a little off from what [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Blob/Blob) and the [File API](https://w3c.github.io/FileAPI/#blob-section) specifies. Particularly, `lastModified` doesn't seem to be part of the specifications, which only call out `type` and `endings`.

Reply:
https://github.com/facebook/react-native/blob/2bf866e4018ea72c1f1c92c806db85378c801fb7/Libraries/Blob/BlobTypes.js#L26